src/index.html

```diff
@@ -67,6 +66,274 @@ <h4 id="title">I'm a software Developer</h4>
+    </div>  
+    <p class="project-note">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+    <div class="project-stack">
+        <ul class="project-tools">
+            <li class="tech-used">HTML</li>
+            <li class="tech-used">Bootstrap</li>
+            <li class="tech-used">Ruby</li>
+        </ul>
+    </div>
+    <button class="see-project">See Project</button>
+  </div>
+  <div class="projects-done">
+    <div class="project-name">
+      <p class="name"> Professional Art</p>
+      <p class="name-data">Printing Data</p>
+    </div>  
+    <p class="project-note">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+    <div class="project-stack">
+        <ul class="project-tools">
+            <li class="tech-used">HTML</li>
+            <li class="tech-used">Bootstrap</li>
+            <li class="tech-used">Ruby</li>
+        </ul>
+    </div>
+    <button class="see-project">See Project</button>
+  </div>
+
+  <div class="projects-done">
+    <div class="project-name">
+      <p class="name"> Professional Art</p>
+      <p class="name-data">Printing Data</p>
+    </div>  
+    <p class="project-note">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+    <div class="project-stack">
+        <ul class="project-tools">
+            <li class="tech-used">HTML</li>
+            <li class="tech-used">Bootstrap</li>
+            <li class="tech-used">Ruby</li>
+        </ul>
+    </div>
+    <button class="see-project">See Project</button>
+  </div>
+
+
+
+  
+  <div class="projects-done">
+    <div class="project-name">
+      <p class="name"> Professional Art</p>
+      <p class="name-data">Printing Data</p>
+    </div>  
+    <p class="project-note">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+    <div class="project-stack">
+        <ul class="project-tools">
+            <li class="tech-used">HTML</li>
+            <li class="tech-used">Bootstrap</li>
+            <li class="tech-used">Ruby</li>
+        </ul>
+    </div>
+    <button class="see-project">See Project</button>
+  </div>
+
+  <!-- Second card -->
+</div>
+
+
+
+
+                
+               
+           </section>
+      
+    </section>
+
+  
+
+    <section class="about">
+        <div class="about_me">
+            <h2 class="about-title">About me</h2>
+            <p class="myNote">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <button class="resume">Get my resume</button>
+          </div>
+          <hr class="stack">
+    </section>
```

Comment:
What you saying is not clear. What do You want me to change. there is nothing in the required change

Reply:
So the only thing I should change here is removing the blank spaces from the html file??

Reply:
- [ ] I suggest that you make all the changes, both **optional** & **required** but the optional changes will not affect your progress to the next milestone. It is  imperative that you make those changes too since you will learn a lot in the process & this knowledge will be very helpful in the journey ahead 😄